index.html

```diff
@@ -88,5 +88,301 @@ <h3>LET’S CONNECT</h3>
+            <li>2015</li>
+          </ul>
+        </div>
+        <p class="work-description text-color-primary margin-top-20">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="work-categories margin-top-12 padding-0">
+          <li><span>html</span></li>
+          <li><span>css</span></li>
+          <li><span>javascript</span></li>
+        </ul>
+        <div>
+          <button type="button" class="button button-enabled">
+          See Project
+        </button>
+        </div>
+      </div>
+      </div>
+    </div>
+  </section>
+  <section class="works">
+    <div class="grid works-grid-container wrap padding-top-0">
+      <div class="flex works-grid-item">
+        <img
+          class="width-100"
+          src="Image/Fitness.png"
+          alt="portfolio screenshot"
+        />
+        <div class="flex work-description-wrapper">
+        <h2 class="section-title margin-top-12">Tonic</h2>
+        <div class="work-period">
+          <ul>
+            <li class="text-color-primary">Canopy</li>
+            <li>Back End Dev</li>
+            <li>2015</li>
+          </ul>
+        </div>
+        <p class="work-description text-color-primary margin-top-20">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="work-categories margin-top-12 padding-0">
+          <li><span>html</span></li>
+          <li><span>css</span></li>
+          <li><span>javascript</span></li>
+        </ul>
+        <div>
+          <button type="button" class="button button-enabled">
+          See Project
+        </button>
+        </div>
+      </div>
+      </div>
+      <div class="flex works-grid-item">
+        <img
+          class="width-100"
+          src="Image/Printing.png"
+          alt="styling screenshot"
+        />
+        <div class="flex work-period-wrapper">
+        <h2 class="section-title margin-top-12">Multi-Post
+          <br> Stories</h2>
+        <div class="work-period">
+          <ul>
+            <li class="text-color-primary">Canopy</li>
+            <li>Back End Dev</li>
+            <li>2015</li>
+          </ul>
+        </div>
+        <p class="work-description text-color-primary margin-top-20">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="work-categories margin-top-12 padding-0">
+          <li><span>html</span></li>
+          <li><span>css</span></li>
+          <li><span>javascript</span></li>
+        </ul>
+        <div>
+          <button type="button" class="button button-enabled">
+          See Project
+        </button>
+        </div>
+      </div>
+      </div>
+    </div>
+    </section>
+      <section class="about-myself padding-top-114">
+      <div class="flex-about-wrap">
+        <div class="container about-section">
```

Comment:
- [OPTIONAL] Kindly make sure to add the correct margin-top to the `about` section in order to look like the provided design 😉 and Please make sure to align it correctly on the page.

   - ### Your design

![wrapp](https://user-images.githubusercontent.com/87197412/185666443-af4d3116-201f-4377-9bc6-a8c0b240b112.PNG)

  - ### Figma design
  
![image](https://user-images.githubusercontent.com/87197412/185666209-afe02737-26c2-43e5-98f1-d8ccd88337e2.png)